app/services/idv/proofer.rb

```diff
@@ -7,30 +7,30 @@ def mock_fallback_enabled?
 
       def resolution_job_class
         if mock_fallback_enabled?
-          require 'identity-idp-functions/proof_resolution_mock'
+          require 'identity_idp_functions/proof_resolution_mock'
           IdentityIdpFunctions::ProofResolutionMock
         else
-          require 'identity-idp-functions/proof_resolution'
+          require 'identity_idp_functions/proof_resolution'
           IdentityIdpFunctions::ProofResolution
         end
       end
 
       def address_job_class
         if mock_fallback_enabled?
-          require 'identity-idp-functions/proof_address_mock'
+          require 'identity_idp_functions/proof_address_mock'
           IdentityIdpFunctions::ProofAddressMock
         else
-          require 'identity-idp-functions/proof_address'
+          require 'identity_idp_functions/proof_address'
```

Comment:
Will we also want to rename `identity_idp_functions` to something else in the future?

Reply:
Yeah I think as part of pass 3, we move the body of the `Proof*` classes inside the job classes (and remove the separate classes)

so then the remaining things in the namespace
- mock clients: just move to top level? Or maybe a `Testing::` namespace? :shrug:
- `S3Helper`, `Timer`, `EncryptionHelper` maybe move to a `Worker::` namespace?

Reply:
👍🏼 👍🏼